packages/models/src/migrations/20230504202005-add-reason.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('activity', 'reason', {
```

Comment:
What's reason used for?

Reply:
Reason for invalidation.  https://discord.com/channels/404673842007506945/1095784856316805342/1105730926295646208

Reply:
Ah got it. 
Low priority but it would be nice to add a description for each column in the model: https://github.com/OriginProtocol/storypoints/blob/main/packages/models/src/models/activity.ts